README.md

```diff
@@ -27,35 +29,74 @@ expirationd.start(job_name, space.id, is_expired, {
 })
 ```
 
+Сustomized version:
+```lua
+expirationd.start(job_name, space.id, is_expired, {
+    -- default is primary key
+    index = 'exp',
+    -- one transaction per batch
+    -- default is false
+    atomic_iteration = true,
+    -- delete data that was added a year ago
+    -- default return nil
+    start_key = function( task )
+        return clock.time() - (365*24*60*60)
+    end,
+    -- delete it from the oldest to the newest
+    -- default is ALL
+    iterator_type = 'GE',
+    -- stop full_scan if delete a lot
+    -- default return true
+    process_while = function( task )
+        if task.args.max_expired_tuples >= task.expired_tuples_count then
+            task.expired_tuples_count = 0
+            return false
+        end
+        return true
+    end,
+    -- this function is default if no other is specified
```

Comment:
this function must return an iterator over the tuples

Let's avoid saying that the listed function is "default" for not confusing users if the actual code will change in the future.